src/pages/Home.js

```diff
@@ -83,26 +83,37 @@ function Header({ carePlan, patient }) {
   );
 }
 
-const HomeContent = ({ carePlan }) => {
-  const tabs = carePlan
-    ? [
-        {
+const getCarePlanTabs = (carePlan) => {
+  const tabOrder = process.env.REACT_APP_HOME_TAB_ORDER ?
+    process.env.REACT_APP_HOME_TAB_ORDER.split(',').map((tabName) => tabName.toLowerCase()) :
```

Comment:
To be on the extra safe side, I'd remove whitespace on either end of the tabName in case someone writes the tab order as `about, goals, tasks`
```suggestion
    process.env.REACT_APP_HOME_TAB_ORDER.split(',').map((tabName) => tabName.toLowerCase().trim()) :
```